vsock_sample/py/vsock-sample.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/local/bin/env python3
+import argparse
+import socket
+import sys
+
+# Client
+class VsockStream:
+    def __init__(self, conn_tmo=5):
+        self.conn_tmo = conn_tmo
+
```

Comment:
PEP8 suggests using 2 blank lines between functions. You should use pycodestyle linter.